src/integrationTest/resources/servicebus/message/supplementary-evidence-example.json

```diff
@@ -1,9 +1,12 @@
 {
   "id": "0192",
   "case_ref": "1539007368674134",
+  "po_box": "BULKSCAN PO BOX",
   "jurisdiction": "BULKSCAN",
   "classification": "SUPPLEMENTARY_EVIDENCE",
   "zip_file_name": "zip-file-test.zip",
+  "delivery_date": "2018-01-01T12:34:56.123Z",
```

Comment:
Are you sure about the date format. as far as I am aware date format is different. Worth checking what ccd would format ccd would expect.

Reply:
Checked with ccd api, supports all `JavaTimeModule` deserialisers so this one too.